src/javascript/app/pages/user/reality_check/reality_check.ui.js

```diff
@@ -101,7 +100,8 @@ const RealityCheckUI = (() => {
     const closePopUp = () => {
         $('#reality_check').remove();
         startSummaryTimer();
-        InterviewPopup.onLoad();
+        // TODO: uncomment below to enable interview popup dialog
```

Comment:
same here

Reply:
I think let's keep it, we will use it in the future, same as before